### PR TITLE
Updated to version 2.1.0

### PR DESCRIPTION
**Pull request by jadsonbr**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

@conda-forge-admin ,please rerende

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {% set name = "pyreportjasper" %}
-{% set version = "1.0.2" %}
+{% set version = "2.1.0" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (Matt-BF) [inline recipe/meta.yaml:9]: As far as I remember, conda-forge discourages using github links instead of pypi

File: recipe/meta.yaml [hunk 2]
```diff
@@ -18,21 +18,21 @@ requirements:
...
     - python
-
-test:
-  imports:
-    - pyreportjasper
-    - test
+    - jpype1
 
 about:
-  home: https://github.com/PyReportJasper/pyreportjasper
```
Comment (Matt-BF) [inline recipe/meta.yaml:30]: Why change this to the docs page instead of the github page?

**Comment by Matt-BF** [comment]
The docker build says `ModuleNotFoundError: No module named 'conda_forge_ci_setup'`. Any idea why this is happening?

File: recipe/meta.yaml (hunk 1 above)
Comment (jadsonbr) [reply recipe/meta.yaml:9]: I am waiting for the acceptance of a request to increase the upload limit for the pypi and until today I have not received a reply, I have not been able to send the new version yet to the pypi.

File: recipe/meta.yaml (hunk 2 above)
Comment (jadsonbr) [reply recipe/meta.yaml:30]: As it is the official documentation page

**Comment by jadsonbr** [comment]
> The docker build says `ModuleNotFoundError: No module named 'conda_forge_ci_setup'`. Any idea why this is happening?

I have no idea why this is happening. Just follow the steps indicated by making a fork and making the necessary changes, not changing anything in relation to conda_forge_ci_setup

**Pull request closed without merging**